projects/packages/jetpack-mu-wpcom/changelog/add-post-migration-task-review-plugins

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: added
+
+Add a new task for plugins review post migration
```

Comment:
Hi @jeherve! Thank you for the answer!

Just to make my question more clear, currently, it's behind a feature flag, so users won't see it for now. I was wondering if we would need it now or when we activate the feature flag. But based on your answer, I suspect we really need the changelog now anyway, correct?

Reply:
The changelog is only visible to folks checking the package's versioning and changelog. It is not directly exposed to customers since site owners do not run just the package. It's consequently something that's more useful for developers leveraging the package (currently only a12s I would imagine). In this situation a changelog entry is useful, even if everything still sits behind a feature flag.